### DISCUSSION TITLE
Collapse or Expande

### DISCUSSION THREAD

**Discussion by Hyfin123** [requarks/wiki#7288] [Help / Questions]
I hope this message finds you well.

I am writing to seek your assistance in creating a Wiki.js documentation site that features an expandable and collapsible menu, similar to the one depicted in the attached image. 

![wiki js](https://github.com/user-attachments/assets/3e4b4469-0524-4fd9-892c-f50ba91061c4)

**Comment by FOCCms** [comment]
In the version 3: https://beta.js.wiki/blog/2023-wiki-js-3-feature-preview-navigation-search
